openedx/features/sketchfab_viewer/static/js/ModelViewer/Components/viewer.js

```diff
@@ -0,0 +1,263 @@
+      <center>
+        <h1 class="m-3">Model Viewer</h1>
+      </center>
+      <h4>Search for images to display in Virtual Reality:</h4>
+
+      
+        <div class="col-6 d-flex justify-content-end">
+          <input
+            type="text"
+            class="form-control"
+            placeholder="e.g. Car"
+            style={{
+              
+
+             
+            }}
+            onChange={this.handleChange}
+          />
+        </div>
+        <div class="col-8">
+          <Button variant="danger" onClick={this.search}>
+            Submit
+          </Button>
+        </div>
+
+
+      <div class="row align-items-center">
+          <iframe
+            title="viewer1"
+            src=""
+            id="api-frame"
+            allow="autoplay; fullscreen; vr"
+            allowvr
+            allowfullscreen
+            mozallowfullscreen="true"
+            webkitallowfullscreen="true"
+            style={{ width: '1733px', height: '990px' }}
+          />
+          <div class="row align-items-end" id="results">
+          <h2>Results:</h2>
+               <Table responsive>
+            <tbody>
+              {this.state.modelsReturned.map(item => (
+                <th key={item[0]}>
+                <CardDeck>
+                <Card>
+                      <Card.Img
+                          variant="top"
+                          onClick={() => callSketch(item[2])}
+                          src={item[1]}
+                          alt={item[0]}
+                          title={item[0]}
+                          style={{height:'250px', width: '250px',  cursor: 'pointer'}}
+                          />
+                        <Card-Body>
+                        <Card-Title>{item[0]}</Card-Title>
+                        </Card-Body>
+                        </Card>
+                        </CardDeck>
+                    
+                      
+
+                </th>
+              ))}
+            </tbody>
+          </Table>
+          <div class="row justify-content-between">
+            <ul class="pagination ">
+                <div class="col-6">
+                  <li class="page-item" key="Previous">
+                    { this.state.previous_page_url != null &&
+                      <Button class="page-link" onClick={() => this.searchByURL(false)}>Prev.</Button>
+                    }
+                      <span class="sr-only">Previous</span>
+                  </li>
+                </div>
+            </ul>
+            <ul class="pagination ">
+                <div class="col-6">
+                  <li class="page-item" key="Next">
+                    { this.state.next_page_url != null &&
+                      <Button class="page-link" onClick={() => this.searchByURL(true)}>Next</Button>
+                    }
+                    <span class="sr-only">Next</span>
+                  </li>
+                </div>
+            </ul>
+          </div>
+
+      </div>

```

Comment:
Please make indentation properly

Reply:
Code is provided by the client themselves.